ui/src/composables/useRestoreUrl.ts

```diff
@@ -86,7 +86,7 @@ export default function useRestoreUrl(options: UseRestoreUrlOptions = {}) {
 
     // Automatically call goToRestoreUrl on mount if needed (equivalent to created() hook)
     onMounted(() => {
-        if (Object.keys(route.query).length === 0 && restoreUrl) {
+        if (restoreUrl) {
```

Comment:
This should probably be `route.query !== defaultQuery && restoreUrl` where 		`defaultQuery` is the filter applied when one first comes to the page. 

As of now, if you have been to this page ever, linking to this page with a URL filter is useless since it will always be replaced.